<launch>
  <param
    name="robot_description"
    textfile="$(find arm_ik)/urdf/arx5_description_ik.urdf" />
    
  <!--node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
      <rosparam param="source_list">[/puppet/joint_right]</rosparam>
  </node-->

  <param name="robot_description_semantic" textfile="$(find arm_ik)/urdf/arx5_description.srdf" />

  <!--node name="hand_eye_tf" pkg="tf" type="static_transform_publisher" 
  args="0.01 0.19 0.65 -0.650 0.647 -0.269 0.292 base_link camera_f_link 50" /-->
  
  <!--手眼标定参数-->
  <!-- <node name="hand_eye_tf" pkg="tf" type="static_transform_publisher" 
  args="-0.01 0.24 0.74 -0.624 0.671 -0.281 0.284 base_link camera_link 100" /> -->

  <!--手眼标定参数，右臂坐标 -->
  <!-- <node name="hand_eye_tf" pkg="tf" type="static_transform_publisher" 
  args="-0.01 0.25 0.6 -0.624 0.671 -0.281 0.284 base_link camera_link 100" /> -->
  <node name="hand_eye_tf" pkg="tf" type="static_transform_publisher" 
  args="-0.02 0.254 0.69 -0.624 0.671 -0.281 0.284 base_link camera_link 100" />

  <!--手眼标定参数，左臂坐标 -->
  <node name="left_hand_eye_tf" pkg="tf" type="static_transform_publisher" 
  args="-0.01 0.58 0.02 0 0 0 base_link base_link_l 100" />


  <node name="green_to_red_tf" pkg="tf" type="static_transform_publisher" 
  args="0.0 -0.09 0.0 0.0 0.0 0.0 grab_link grab_red_link 100" />

  <!-- <node name="grab_step_tf" pkg="tf" type="static_transform_publisher" 
  args="0 0 0.06 0 0 0 0 grab_link grab_step_link 100" /> -->

  <node
    name="robot_state_publisher"
    pkg="robot_state_publisher"
    type="robot_state_publisher" />
</launch>
